bitnami/thanos/Chart.yaml

```diff
@@ -28,4 +28,4 @@ name: thanos
 sources:
   - https://github.com/bitnami/bitnami-docker-thanos
   - https://thanos.io
-version: 3.14.1
+version: 3.14.2
```

Comment:
As this PR adds multiple settings, could you please replace the version with a minor version bump `3.15.0`?

Reply:
Bumped to 3.15.0